Description: Fix a lintian warning about a typo in the man page
 spelling-error-in-manpage usr/share/man/man1/htop.1.gz
Author: Daniel Lange <dl.ml1@usrlocal.de>
---
--- a/htop.1.in
+++ b/htop.1.in
@@ -10,7 +10,7 @@ Htop is a free (GPL) ncurses-based proce
 .LP
 It is similar to top, but allows you to scroll vertically and horizontally,
 so you can see all the processes running on the system, along with their full
-command lines, as well as viewing them as a process tree, selecting mutiple
+command lines, as well as viewing them as a process tree, selecting multiple
 processes and acting on them all at once.
 .LP
 Tasks related to processes (killing, renicing) can be done without
